app/src/components/searchWidget/date-input/expandedDatePanel.tsx

```diff
@@ -79,24 +83,33 @@ export class ExpandedDatePanel extends React.Component<StateChangedProps, any> {
         }
     }
 
+    changeColor() {
+        this.setState({
+            newButton: !this.state.oldButton,
+            clicked: true
+        });
+    }
+
     render() {
         const { from, to } = this.state;
         return (
             <div className="date-panel-expanded">
-                <div className="top-toogler">
+                <ToggleButtonGroup className="datetoggle" name="options" defaultValue={1}>
                     <div
                         className="dates-selector middle-text"
                         onClick={() => this.selectPanel(datePanelTypes.SPECIFIC_DATES)}
                     >
-                        <div>Specific Dates</div>
+                        <Button onClick={this.changeColor.bind(this)}>Specific Dates</Button>
```

Comment:
Please move this onClick actions to .selectPanel() method

Reply:
Selected button should also keep selected after you select-close-open that panel. doesn't look like it's covered

Reply:
maybe set color for a button in .saveSelectedPanel()

Reply:
yes I'm trying to set it in saveSelectedPanel()

Reply:
@rmit-s3668804-Dzmitry-Kakaruk I could not figure out what exact type of buttons is being used for this part by webjet. Any leads about that would be helpful.

Reply:
@rmit-s3668804-Dzmitry-Kakaruk and @mmo-mikhail Fixed it to look as close as webjet's and the state now works.